optuna/cli.py

```diff
@@ -382,6 +382,30 @@ def take_action(self, parsed_args: Namespace) -> int:
         return 0
 
 
+class _StudyNames(_BaseCommand):
+    """Get all study names stored in a specified storage"""
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "-f",
+            "--format",
+            type=str,
+            choices=("json", "table", "yaml"),
+            default="table",
```

Comment:
We discussed the default format and decided to output study names per line like below. Could you update the PR?

```
$ optuna study-names --storage ...
study_1
study_2
```

Reply:
I updated so that `study-names` output is default to `value` format. PTAL